testsuite/features/secondary/min_ansible_control_node.feature

```diff
@@ -10,6 +10,19 @@ Feature: Operate an Ansible control node in a normal minion
   Scenario: Pre-requisite: Deploy test playbooks and inventory file
     When I deploy testing playbooks and inventory files to "sle_minion"
 
+  @susemanager
+  Scenario: Pre-requisite: Subscribe SUSE minions to SLE-Module-Python3-15-SP4-Pool for x86_64
+    Given I am on the Systems overview page of this "sle_minion"
+    When I follow "Software" in the content area
+    And I follow "Software Channels" in the content area
+    And I check "SLE-Module-Python3-15-SP4-Pool for x86_64" by label
+    And I check "SLE-Product-SLES15-SP4-LTSS-Updates for x86_64" by label
```

Comment:
From what I see in other parts of our testsuite, it does not seem we need `by label`.

See for instance in:
```
  Scenario: Cleanup: select the channels of the SLES minion as before
    When I follow "Software" in the content area
    And I follow "Software Channels" in the content area
    And I wait until I do not see "Loading..." text
    And I check radio button "SLE-Product-SLES15-SP4-Pool for x86_64"
    And I wait until I see "SLE15-SP4-Installer-Updates for x86_64" text
    And I include the recommended child channels
    And I check "SLE-Module-DevTools15-SP4-Pool for x86_64"
    And I check "Fake-RPM-SUSE-Channel"
    And I check "SLE-Product-SLES15-SP4-LTSS-Updates for x86_64"
```